### PR TITLE
Add insiders badges

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Add insiders badges

### CODE REVIEW

File: app/commands/user/insiders_status/update.rb [hunk 1]
```diff
@@ -26,8 +26,8 @@ def update_eligible_lifetime
     when :active_lifetime
       user.update(insiders_status: :active_lifetime)
     when :active
-      user.update(insiders_status: :active_lifetime)
-      User::Notification::Create.(user, :joined_lifetime_insiders)
+      user.update(insiders_status: :eligible_lifetime)
+      User::InsidersStatus::Activate.(user)
```
Comment (ErikSchierboom) [inline app/commands/user/insiders_status/update.rb]: I'm using `User::InsidersStatus::Activate` to actually convert the `active` status to `active_lifetime`. That way we can have all activation-related logic (like awarding the badge) in one place

File: app/images/icons/insider.svg
Comment (ErikSchierboom) [inline app/images/icons/insider.svg:1]: TODO

File: app/images/icons/original-insider.svg
Comment (ErikSchierboom) [inline app/images/icons/original-insider.svg:1]: TODO

File: app/models/badges/insider_badge.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module Badges
+  class InsiderBadge < Badge
+    seed "Insider",
+      :ultimate,
```
Comment (ErikSchierboom) [inline app/models/badges/insider_badge.rb:4]: Not sure if this should be legendary or ultimate 🤷

File: app/models/badges/insider_badge.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      'insider',
+      'Member of Exercism Insiders'
+
+    def award_to?(user) = VALID_INSIDERS_STATUSES.include?(user.insiders_status)
+    def send_email_on_acquisition? = true
+
+    VALID_INSIDERS_STATUSES = %i[active active_lifetime].freeze
```
Comment (ErikSchierboom) [inline app/models/badges/insider_badge.rb:11]: The majority of people will get this via the `active` status, but I added `active_lifetime` for those initial users that immediately become lifetime insiders.

File: app/models/badges/original_insider_badge.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module Badges
+  class OriginalInsiderBadge < Badge
+    seed "Original Insider",
+      :legendary,
+      'original-insider',
+      'One of the Original Insiders'
```
Comment (ErikSchierboom) [inline app/models/badges/original_insider_badge.rb:6]: Feel free to change the copy

File: .gitignore [hunk 5]
```diff
@@ -31,6 +31,7 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.yarn/
```
Comment (ErikSchierboom) [inline .gitignore:34]: I updated my nodejs and got the latest yarn version which handles things a bit differently. That's for another PR sometime though

File: app/models/badges/original_insider_badge.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    seed "Original Insider",
+      :legendary,
+      'original-insiders',
+      'One of the Original Insiders'
+
+    def award_to?(user) = ORIGINAL_INSIDER_HANDLES.include?(user.handle.downcase)
+    def send_email_on_acquisition? = true
+
+    # TODO: update original insider handles just before launching
+    ORIGINAL_INSIDER_HANDLES = %w[kytrinyx ihid erikschierboom].freeze
```
Comment (iHiD) [inline app/models/badges/original_insider_badge.rb:12]: This is going to be a long list!

Comment (ErikSchierboom) [reply app/models/badges/original_insider_badge.rb:12]: Yep!

**Pull request merged**